package.json

```diff
@@ -83,7 +83,7 @@
         "sinon": "^14.0.0",
         "supertest": "4.0.2",
         "typedoc": "^0.22.5",
-        "typescript": "3.8.3"
+        "typescript": "4.2"
```

Comment:
i thought you had changed this version in the previous PR.. this change again? Im so confused.